test_runner/src/main/kotlin/ftl/run/platform/android/CreateAndroidTestContext.kt

```diff
@@ -75,11 +76,14 @@ private fun InstrumentationTestContext.downloadApks(): InstrumentationTestContex
     test = test.downloadIfNeeded()
 )
 
-private fun InstrumentationTestContext.calculateShards(
+private fun InstrumentationTestContext.calculateShardsNormally(
     args: AndroidArgs,
     testFilter: TestFilter = TestFilters.fromTestTargets(args.testTargets, args.testTargetsForShard)
 ): InstrumentationTestContext = ArgsHelper.calculateShards(
-    filteredTests = getFlankTestMethods(testFilter),
+    filteredTests = getFlankTestMethods(
+        testFilter = testFilter,
+        parameterizedTests = args.parameterizedTests ?: FlankDefaults.DEFAULT_PARAMETERIZED_TESTS
```

Comment:
`AndroidGcloudConfig` has a default value for `parameterizedTests` therefor I think `parameterizedTests` should not be nullable (in `AndroidArgs`) To make it happen you will also need to make small change in `CreateAndroidArgs`:
`parameterizedTests = gcloud.parameterizedTests` -> `parameterizedTests = gcloud::parameterizedTests.require()`

With the above changes, we can get rid of 'elvis defaults' calls in other places. I think it's worth changing, we will have only one place to update if we decide to change a default value.
Let me know what you think

Reply:
good spot thanks for the advice and its been implemented :)